src/core/interfaces-writer/SingleFileInterfacesWriter.ts

```diff
@@ -1,11 +1,11 @@
 import * as path from "path";
-import {ApiDoc2InterfaceParameters} from "../ApiDoc2Interface";
-import {ConverterResult} from "../endpoint-converter/ApiDocToInterfaceConverter";
-import {writeFileToPath} from "../utils/FsUtils";
-import {stringifyAllInterfaces} from "../utils/WriterUtils";
-import {InterfacesWriter} from "./InterfacesWriter";
+import { ApiDoc2InterfaceParameters } from "../ApiDoc2Interface";
```

Comment:
This code-style is controversial, can we configure linter so we don't have to place spaces inside brackets?
If no, let's disable this and add prettier instead of all code-style linter rules

Reply:
We can force not using spaces inside curly brackets or disable this rule check at all. I think we can use prettier alongside with linter or create our own set of rules based on airbnb's, which we are currently extending of.

Reply:
Ok, let's force not using spaces inside curly brackets for now

Reply:
Done